Index: src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy b/src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy
--- a/src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy	(revision 2972238f34e6162bc57cd49e35887752cd9eb232)
+++ b/src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy	(revision 6d89e2eaea898e96f697f5400abde70d932df982)
@@ -848,8 +848,8 @@
     }
 
     @NonCPS
-    def sortTestSteps(steps) {
-        return steps?.sort(false) { it.orderId }
+    def sortTestSteps(def testSteps) {
+        return testSteps?.sort(false) { it.orderId }
     }
 
     String createSSDS(Map repo = null, Map data = null) {
Index: test/groovy/org/ods/orchestration/usecase/LeVADocumentUseCaseSpec.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test/groovy/org/ods/orchestration/usecase/LeVADocumentUseCaseSpec.groovy b/test/groovy/org/ods/orchestration/usecase/LeVADocumentUseCaseSpec.groovy
--- a/test/groovy/org/ods/orchestration/usecase/LeVADocumentUseCaseSpec.groovy	(revision 2972238f34e6162bc57cd49e35887752cd9eb232)
+++ b/test/groovy/org/ods/orchestration/usecase/LeVADocumentUseCaseSpec.groovy	(revision 6d89e2eaea898e96f697f5400abde70d932df982)
@@ -1631,7 +1631,10 @@
             ]]
 
             when:
-            def ordered = testIssue.steps?.sort { it.orderId }
+            LeVADocumentUseCase leVADocumentUseCase = new LeVADocumentUseCase(null, null, null,
+                null, null, null, null, null, null, null,
+                null, null)
+            def ordered = leVADocumentUseCase.sortTestSteps(testIssue.steps)
 
             then:
             ordered.get(0).orderId == 1
